src/app/components/MostRead/index.tsx

```diff
@@ -145,6 +145,9 @@ const MostRead = ({
       optimizely,
       optimizelyMetricNameOverride: 'most_read',
     }),
+    ...(isLite && {
+      liteSiteUrl: 'liteSiteUrl',
```

Comment:
I know this is just a spike, but what is this value supposed to be - the .lite version of the page it's on?